packages/ui/react-ui/src/Button.tsx

```diff
@@ -3,28 +3,26 @@ import React, { CSSProperties, FC, MouseEvent, ReactElement } from 'react';
 export interface ButtonProps {
     className?: string;
     disabled?: boolean;
-    endIcon?: ReactElement;
     onClick?: (e: MouseEvent<HTMLButtonElement>) => void;
     startIcon?: ReactElement;
+    endText?: string;
```

Comment:
Hmm this is a breaking API change for this component because it removes the `endIcon` prop. We should really just evaluate the Torus changes on their own.